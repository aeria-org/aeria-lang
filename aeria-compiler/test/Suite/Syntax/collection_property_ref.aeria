collection CollectionPropertyRefTest2 {
}

collection CollectionPropertyRefTest {
  properties {
    prop   CollectionPropertyRefTest2
  }
}
