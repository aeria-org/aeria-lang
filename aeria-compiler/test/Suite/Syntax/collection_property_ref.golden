{
  "kind": "Program",
  "collections": [
    {
      "name": {
        "value": "collectionpropertyreftest2",
        "kind": "CollectionName"
      },
      "kind": "Collection"
    },
    {
      "properties": [
        {
          "type": {
            "ref": {
              "value": "collectionpropertyreftest2",
              "kind": "CollectionName"
            },
            "kind": "PRef"
          },
          "name": {
            "value": "prop",
            "kind": "PropertyName"
          },
          "kind": "Property",
          "attributes": []
        }
      ],
      "name": {
        "value": "collectionpropertyreftest",
        "kind": "CollectionName"
      },
      "kind": "Collection"
    }
  ]
}
